generated/nidigital/nidigital/session.py

```diff
@@ -3154,14 +3699,19 @@ def _load_timing(self, file_path):
     def read_sequencer_flag(self, flag):
         r'''read_sequencer_flag
 
-        TBD
+        Reads the state of a pattern sequencer flag. Use pattern sequencer flags to coordinate execution between the pattern sequencer and a runtime test program.
 
         Args:
-            flag (enums.SequencerFlag):
+            flag (enums.SequencerFlag): The pattern sequencer flag you want to read.
+
+                -   SequencerFlag.FLAG0 ("seqflag0"): Writes pattern sequencer flag 0.
+                -   SequencerFlag.FLAG1 ("seqflag1"): Writes pattern sequencer flag 1.
+                -   SequencerFlag.FLAG2 ("seqflag2"): Writes pattern sequencer flag 2.
+                -   SequencerFlag.FLAG3 ("seqflag3"): Writes pattern sequencer flag 3.
```

Comment:
"Writes" --> "Reads"

Reply:
Note, this is how it appears in the C API Reference.

Reply:
Done

Reply:
Filed internal bug (1270694)